src/client.ts

```diff
@@ -281,6 +283,22 @@ export interface IPerxUserService {
    * @param perPage desinate the page size
    */
   queryLoyaltyTransactionsHistory(userToken: string, page: number, perPage: number): Promise<PerxLoyaltyTransactionsHistoryResponse>
+
+  /**
+   * Query all merchants from Perx
+   * @param  {boolean} favorite
+   * @param  {number} perPage
+   * @returns
+   */
+  listAllMerchants(userToken: string, favorite: boolean, perPage: number): Promise<PerxMerchantResponse>
```

Comment:
Please make sure favorite can accept undefined value.
Please also add page: number to the argument list.